engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/UniverseSetupScreen.java

```diff
@@ -315,6 +331,17 @@ public void setEnvironment(UniverseWrapper wrapper) {
         }
     }
 
+    public int findIndex(List<WorldSetupWrapper> worldsList,String worldName) {
+        for(int i = 0 ;i<worldsList.size();i++) {
+            WorldSetupWrapper random = worldsList.get(i);
```

Comment:
Better variable name than `random` please - how is it random anyway? It is the specific world in each location in the list processed by the `for`